hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileCreateHandlerTest.java

```diff
@@ -209,7 +210,7 @@ void handleWorksAsExpected() {
         assertEquals(1_234_567L, actualFile.expirationTime());
```

Comment:
We've decided to only use AssertJ for assertions. All of these assertions should be changed to use AssertJ.

Reply:
if that's the case I think we need to change the all-unit test in all services. most of the services use asserts from junit

Reply:
can I open a separate PR for it later on depends on the priority.

Reply:
no problem but can we do it as part of a separate PR.